README.md

```diff
@@ -2,16 +2,33 @@
 
 ![image](https://user-images.githubusercontent.com/28736511/51742581-80aa0c80-20dd-11e9-8ed9-b591a79442a3.png)
 
-A curated list of NILM (also known as Energy Disaggregation) resources. Inspired by [awesome-deep-vision](https://github.com/kjw0612/awesome-deep-vision), [awesome-public-datasets](https://github.com/awesomedata/awesome-public-datasets), [awesome-adversarial-machine-learning](https://github.com/yenchenlin/awesome-adversarial-machine-learning), [awesome-deep-learning-papers](https://github.com/terryum/awesome-deep-learning-papers), [awesome-meta-learning](https://github.com/dragen1860/awesome-meta-learning/) and [awesome-architecture-search](https://github.com/markdtw/awesome-architecture-search).
+A curated list of NILM (also known as Load Disaggregation) resources. Inspired by [awesome-deep-vision](https://github.com/kjw0612/awesome-deep-vision), [awesome-public-datasets](https://github.com/awesomedata/awesome-public-datasets), [awesome-adversarial-machine-learning](https://github.com/yenchenlin/awesome-adversarial-machine-learning), [awesome-deep-learning-papers](https://github.com/terryum/awesome-deep-learning-papers), [awesome-meta-learning](https://github.com/dragen1860/awesome-meta-learning/) and [awesome-architecture-search](https://github.com/markdtw/awesome-architecture-search).
```

Comment:
I wonder why you think load disaggregation is a better expression than energy disaggregation.

Reply:
Just an opinion. I think load disaggregation makes more sense than energy disaggregation. Motivation: power engineers refer to it as load disaggregation. Feel free to ignore this suggestion.